argo_rollouts/using-cluster-scoped-rollouts-instance-to-manage-rollouts-resources.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="using-cluster-scoped-rollouts-instance-to-manage-rollouts-resources"]
+= Using a cluster-scoped rollouts instance to manage rollouts resources
+context: using-cluster-scoped-rollouts-instance-to-manage-rollouts-resources
+
+toc::[]
+
+By default, Argo rollouts supports the cluster-scoped mode of installation for Argo rollouts custom resources (CRs). This mode of installation uses the `CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES` environment variable to specify a list of namespaces that can be used to manage the rollout resources. 
```

Comment:
Can add a line to the end of first paragraph: However, by default, if `CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES` is empty, you can still create cluster-scoped Argo Rollouts installs in the `openshift-gitops` namespace.

Reply:
Fixed.